android/AndroidManifest.xml

```diff
@@ -13,7 +13,7 @@
     <!-- The following comment will be replaced upon deployment with default features based on the dependencies of the application.
          Remove the comment if you do not require these default features. -->
     <!-- %%INSERT_FEATURES -->
-    <uses-feature android:name="android.hardware.microphone" android:required="true"/>
+    <!--<uses-feature android:name="android.hardware.microphone" android:required="true"/>-->
```

Comment:
That's not really good. Either remove it completely or leave it in.

Reply:
Agreed.
@henkdegroot, @chigkim  I've removed this completely now (it's still recoverable via git history). Leaving it in is not possible based on the history (it causes duplicates).